runtime/gc_modron_standard/ContinuationObjectBufferStandard.cpp

```diff
@@ -102,3 +105,36 @@ MM_ContinuationObjectBufferStandard::reinitializeForRestore(MM_EnvironmentBase *
 	return true;
 }
 #endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
+
+void
+MM_ContinuationObjectBufferStandard::iterateAllContinuationObjects(MM_EnvironmentBase *env)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+	MM_HeapRegionDescriptorStandard *region = NULL;
+	GC_HeapRegionIteratorStandard regionIterator(extensions->heapRegionManager);
+
+	env->_currentTask->synchronizeGCThreads(env, UNIQUE_ID);
```

Comment:
put a comment here why we need it (i believe to make sure that previous pruning phase of continuation list is complete)